src/coreclr/gc/gcpriv.h

```diff
@@ -4238,21 +4236,166 @@ class gc_heap
     struct dynamic_heap_count_data_t
     {
         static const int sample_size = 3;
+        static const int recorded_tcp_array_size = 64;
 
         struct sample
         {
             uint64_t    elapsed_between_gcs;    // time between gcs in microseconds (this should really be between_pauses)
             uint64_t    gc_pause_time;          // pause time for this GC
             uint64_t    msl_wait_time;
+            size_t      gc_survived_size;
```

Comment:
```suggestion
            size_t      gc_survived_size;       // total survived size across all relevant generations for this GC
```
i.e., it's -not- gen0 to be consistent in what is being recorded